docs/pages/localization/hijri.mdx

```diff
@@ -0,0 +1,41 @@
+import Ad from '_shared/Ad';
```

Comment:
I think there is no way to make additional example for hijiri. We can make same page for both. 
Just title it like `Additional calendar systems` or smth. Thanks for your work